src/components/shared/ManagedTree.js

```diff
@@ -68,14 +69,36 @@ class ManagedTree extends Component {
     }
   }
 
-  setExpanded = (item: Item, isExpanded: boolean) => {
+  setExpanded = (item: Item, isExpanded: boolean, isRecursive: boolean) => {
```

Comment:
i'm thinking something like `shouldIncludeChildren` feels better and clearer then `isRecursive`